### PR TITLE
improve/argocd

### PR DESCRIPTION
**Pull request by cutezhangq**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it:**
Improve working-with-argocd

**Which issue(s) this PR fixes: **
Fixes #

**Special notes for your reviewer:**

**Does this PR introduce a user-facing change?:**
NONE

### CODE REVIEW

**Comment by RainbowMango** [comment]
@cutezhangq Can you use the PR template?

**Comment by cutezhangq** [comment]
@RainbowMango OK, I've made some changes.

File: docs/working-with-argocd.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Working with Argo CD
 
-This topic walks you through how to use the [Argo CD](https://github.com/argoproj/argo-cd/) to manage your workload 
-`across clusters` with `Karmada`.
+This topic walks you through how to use [Argo CD](https://github.com/argoproj/argo-cd/) to manage your workload `across clusters` with `Karmada`.
```
Comment (RainbowMango) [inline docs/working-with-argocd.md:3]: Why do you think we need to put the content in one line?

File: docs/working-with-argocd.md [hunk 2]
```diff
@@ -21,9 +20,9 @@ member3   v1.19.1   Pull   True    17h
 ```
 
 ## Registering Karmada to Argo CD
-This step registers Karmada control plane to Argo CD.
+This step registers the Karmada control plane to Argo CD.
 
-First list the contexts of all clusters in your current kubeconfig:
+First, list the contexts of all clusters in your current kubeconfig:
```
Comment (RainbowMango) [inline docs/working-with-argocd.md:25]: I'm not sure about it. @Tingtal

**Comment by RainbowMango** [comment]
@cutezhangq Can you respond to the questions I made above?

File: docs/working-with-argocd.md [hunk 3]
```diff
@@ -73,24 +71,21 @@ You can sync your applications via UI by simply clicking the SYNC button and fol
 
 ![](./images/argocd-sync-apps.png)
 
-More details please refer to [argocd guide: sync the application](https://argo-cd.readthedocs.io/en/stable/getting_started/#7-sync-deploy-the-application).
+For more details, please refer to the [Argo CD guide: sync the application](https://argo-cd.readthedocs.io/en/stable/getting_started/#7-sync-deploy-the-application).
 
 ## Checking Apps Status
-For deployment running in more than one clusters, you don't need to create applications for each 
-cluster. You can get the overall and detailed status from one `Application`.
+For deployment running in more than one clusters, you don't need to create applications for each cluster. You can get the overall and detailed status from one `Application`.
```
Comment (Tingtal) [inline docs/working-with-argocd.md:77]:
```suggestion
For a Deployment running in more than one cluster, you don't need to create applications for each cluster. You can get the overall and detailed status from one `Application`.
```

File: docs/working-with-argocd.md (hunk 2 above)
Comment (Tingtal) [reply docs/working-with-argocd.md:25]: I'm good with this change.

File: docs/working-with-argocd.md [hunk 4]
```diff
@@ -73,24 +71,21 @@ You can sync your applications via UI by simply clicking the SYNC button and fol
...
+The `svc/guestbook-ui`, `deploy/guestbook-ui` and `propagationpolicy/guestbook` in the middle of the picture are the resources created by the manifest in the forked repo. And the `resourcebinding/guestbook-ui-service` and `resourcebinding/guestbook-ui-deployment` on the right of the picture are the resources created by Karmada.
 
 ### Checking Detailed Status
 You can obtain the Deployment's detailed status by `resourcebinding/guestbook-ui-deployment`. 
 
 ![](./images/argocd-status-resourcebinding.png)
 
 ### Checking Aggregated Status
-You can obtain the aggregated status of the Deployment from UI by `deploy/guestbook-ui`.
+You can obtain the aggregated status of the deployment from UI using `deploy/guestbook-ui`.
```
Comment (Tingtal) [inline docs/working-with-argocd.md:89]: Not sure which style you prefer for describing a stateless app.
Sometimes you capitalize **d**, sometime you don't. 
From my point of view, I will stick to **D**eployment.

**Comment by RainbowMango** [comment]
/close
As the docs have been moved to the website repo. 
If this is still needed, please send a PR on the website repo.

**Pull request closed without merging**